resources/migrations/000_migrations.yaml

```diff
@@ -11932,21 +11932,6 @@ databaseChangeLog:
                   constraints:
                     nullable: false
 
-  - changeSet:
```

Comment:
Is it sound to remove a migration like this? Should it be replaced with another that drops these columns if found?

It's cool if this is fine -- I've just learned to be wary of deploying schema changes.

Reply:
that's a good call.

Hi @flamber, do you think it's okay to just remove the migration like this? or a drop changeset is safer?

Reply:
I would probably not remove the migration in this revert, otherwise it should create a drop changeset, so instances with this changeset won't have problems in the future. I'll defer to @camsaul

Reply:
I kept the migration and added a new migration to drop the column.